contracts/Arbitrum_SpokePool.sol

```diff
@@ -110,7 +110,8 @@ contract Arbitrum_SpokePool is SpokePoolInterface, SpokePool {
         emit WhitelistedTokens(_l2Token, _l1Token);
     }
 
-    // l1 addresses are transformed during l1->l2 calls. See https://developer.offchainlabs.com/docs/l1_l2_messages#address-aliasing for more information.
+    // l1 addresses are transformed during l1->l2 calls. See https://developer.offchainlabs.com/docs/l1_l2_messages#address-aliasing for more information. //todo: expalin why unchecked and why it's not imported from the
```

Comment:
is the goal on these just to document these todos? did you take this from: https://github.com/across-protocol/across-smart-contracts-v2/pull/45